### ISSUE TITLE
Running BCB & GGNN, met exit code 139

### ISSUE THREAD

**Issue by sqhan-Husky** [jacobwwh/graphmatch_clone#1]
Hi, l recently read your paper about Code Clones Detection with GNN. I appreciated that you opensource your codes and l have learned from them.  
But when I ran ggnn_bcb.py without any modification, l got wrong with "Process finished with exit code 139" at GlobalAttention Step (line 3).  
`x = self.ggnnlayer(x, edge_index)  `
`batch=torch.zeros(x.size(0),dtype=torch.long).to(self.device)  `
`hg=self.pool(x,batch=batch)`  

It seems that one array access out of bounds when doing softmax.  l don't know whether it went wrong when you did experiments.

Another question is, l found that you got edge_attr information when creating the fa-ast but didn't use it in the ggnn layer.

I hope to hear from your replies, thank you!

**Comment by sqhan-Husky** [comment]
OK, I have found the reason yet.  Correct torch environment!
